nvflare/app_opt/xgboost/histogram_based_v2/proto/federated.proto

```diff
@@ -1,26 +1,30 @@
 /*!
- * Copyright 2022 XGBoost contributors
- * needs to match file in https://github.com/dmlc/xgboost/blob/v2.0.3/plugin/federated/federated.proto
```

Comment:
do we want to add this comment here for reference,

something like:

* needs to match file in https://github.com/dmlc/xgboost/blob/v2.11/plugin/federated/federated.proto ?

Reply:
This file is copied from XGBoost. Shouldn't make manual changes.